### PR TITLE
Mobile - add direct schedule clinic endpoint

### PR DESCRIPTION
**Pull request by aherzberg**
## Description of change

Add direct schedule clinic endpoint

## Original issue(s)
https://app.zenhub.com/workspaces/va-mobile---backend-61d75d03a7505500147ec7f6/issues/department-of-veterans-affairs/va-mobile-app/3319

## Things to know about this PR

Note that the upstream service does allow for pagination but seems unnecessary since I've only ever seen 3-4 clinics show up at a time and I haven't seen any pagination for the web flow.

### CODE REVIEW

**Comment by aherzberg** [comment]
Realized I forgot to remove the clinics field from the facilities info endpoint. will re-open when that's been added.

**Comment by aherzberg** [comment]
just did clinics removal from facilities-info in another PR to speed up that being merged since FE is going to start integrating as soon as it gets merged.

**Review by jperk51 — COMMENTED** [review]
Do clinics not have id? I see here we are only sending the name back. Is that sufficient for a user to select a given clinic?

**Comment by aherzberg** [comment]
> Do clinics not have id? I see here we are only sending the name back. Is that sufficient for a user to select a given clinic?

I was utilizing the id field that's a sibling to attributes field rather then having another id field within attributes.

File: modules/mobile/spec/support/schemas/clinic.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "data": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [
+          "id",
+          "type",
+          "attributes"
+        ],
+        "properties": {
```
[the remark this answers is not in the document]
Comment (aherzberg) [reply modules/mobile/spec/support/schemas/clinic.json:17]: oops. Good catch.

**Pull request merged**